google-bigquery/1.0/modules/ROOT/pages/google-bigquery-connector-examples.adoc

```diff
@@ -0,0 +1,1182 @@
+= Google BigQuery Connector 1.0 Examples - Mule 4
+
+The following examples show several Mule flows for Google BigQuery Connector:
+
+* <<get-job>>
+* <<insert-job>>
+* <<delete-job>>
+* <<get-job-result>>
```

Comment:
```suggestion
* <<get-query-result>>
```
I found out there are references to **Get Query Result** as **Get Job Result**, I will be updating this into our demos/readme/etc to match the suggestions here